### PR TITLE
Duplicate of line 51

### PR DESCRIPTION
**Pull request by Fale**

### CODE REVIEW

File: ez_setup/__init__.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 This file can also be run as a script to install or upgrade setuptools.
 """
-import sys
+
 DEFAULT_VERSION = "0.6c7"
 DEFAULT_URL = ("http://pypi.python.org/packages/"
                "%s/s/setuptools/" % sys.version[:3])
```
Comment (pypingou) [inline ez_setup/__init__.py]: Except that it's the line 51 that should be removed, otherwise this line will fail ^ :)

Comment (ralphbean) [inline ez_setup/__init__.py]: Agreed!  :)  @fale, can you update?

**Comment by pypingou** [comment]
ping ?

**Comment by pypingou** [comment]
@Fale did you see the comment?

**Comment by ralphbean** [comment]
Fixed in 6a90c33.

**Pull request closed without merging**